assets/javascripts/discourse/components/chat-composer-upload.js

```diff
@@ -0,0 +1,19 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { IMAGES_EXTENSIONS_REGEX } from "discourse/lib/uploads";
+
+export default Component.extend({
+  IMAGE_TYPE: "image",
+
+  tagName: "",
```

Comment:
I would use `span.chat-upload` here, and remove the wrapping span from the `hbs` template.